src/autopas/LogicHandler.h

```diff
@@ -1289,7 +1289,7 @@ std::tuple<std::optional<std::unique_ptr<TraversalInterface>>, bool> LogicHandle
   // Check if the container supports the traversal
   const auto allContainerTraversals = compatibleTraversals::allCompatibleTraversals(conf.container);
   if (allContainerTraversals.find(conf.traversal) == allContainerTraversals.end()) {
-    AutoPasLog(DEBUG, "Configuration rejected: Container {} does not support the traversal {}.", conf.container,
+    AutoPasLog(TRACE, "Configuration rejected: Container {} does not support the traversal {}.", conf.container,
```

Comment:
Adding onto this: I would be more in favour of this if we had container independent traversal names e.g. c08 applies to Linked Cells, Verlet List Cells, Pairwise Verlet Lists.

Could you elaborate by what you meant by
> For me this was a very important message while building the tuner and logic handler 😅 .

If this was purely a useful message in the construction of the logic, I don't think that this is a reason to keep this, even just as a `TRACE` message. If however, this message could provide a useful way of pointing out bugs, I'm happy with this as a `TRACE` message. Unless my reasoning is incorrect in how many times this will be printed, I very strongly would not like this at `DEBUG` level.

Reply:
> Adding onto this: I would be more in favour of this if we had container independent traversal names e.g. c08 applies to Linked Cells, Verlet List Cells, Pairwise Verlet Lists.

As everything is implemented now I'm very much against that. Each traversal is implemented for exactly one container. You can't just take the same traversal and apply it to a different container. Thus it would actually make more sense to remove the container option and deduce it from the traversal, but it is very useful to group the traversal options. Also, when you analyze runs, it is very convenient to just plot traversals and get the split by containers implicitly.

> Could you elaborate by what you meant by
> > For me this was a very important message while building the tuner and logic handler 😅 .

Debug mode as it is in the current master shows very nicely the whole tuning logic. What configurations are there, what is selected / rejected, what are tuning starts doing. When you want to track what exactly happens this is super useful. E.g. during debugging. Hence `DEBUG` log.

> This gets very spammy

Wait, does it? Actually I'd expect this to basically never happen because configurations where container and traversal don't match would not be constructed in the first place. I just ran the following simulation and this was not printed once (compiled with `AUTOPAS_MIN_LOG_LEVEL=TRACE`):

```bash
md-flexible \
  --container all \
  --traversal all \
  --log-level debug \
  --tuning-phases 1
```

Thus I'd argue this should probably be promoted to `WARN`

Reply:
Makes sense. Done in 243514f8